tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -92,13 +92,14 @@ def __init__(self, api_client):
                 is_processed=True,
```

Comment:
How did the autoformatting of this file happen? If you autoformat it, please use black.

Reply:
formatted it with black again